README.md

```diff
@@ -12,28 +15,39 @@
 </div>
 </div>
 
+<!-- ---------- Description ---------- -->
 ## Features
 
 - [x] 30 supported languages
```

Comment:
Almost no but we can note "more than 150 supported languages", that is more reasonable *_^
What do you think about it?

Reply:
If you prefer, I'll write: "Almost all languages are supported"

Reply:
I would be okay with 150+, in the end we should probably differ between the libre and the normal version here, e.g. `150+ supported languages (30+ for the libre version)`, although this might be too long in comparison to the other feature summaries!

Reply:
It could be written like this (well, maybe):
- [x] more than 150 supported languages _(standard version)_
- [x] 30 supported languages _(libre version)_